source/MissionPanel.cpp

```diff
@@ -43,6 +43,26 @@ using namespace std;
 
 namespace {
 	const int SIDE_WIDTH = 280;
+	
+	// Checks if a missions involves the provided system.
```

Comment:
`Determine if the given mission uses the given system`

or at least `a` -> `this` and drop the s on `missions`